### DISCUSSION TITLE
Neper tests FAILED

### DISCUSSION THREAD

**Discussion by zirongzhai** [neperfepx/neper#443] [Q&A] [answered]
Hi,

I got stuck when doing ctest for Neper. please see the following descriptions:

_319/319 Test #319: V/vertrs .........................   Passed    3.31 sec

88% tests passed, 37 tests failed out of 319

Total Test time (real) = 605.30 sec

The following tests FAILED:
        124 - M/cl (Failed)
        125 - M/cl_expr (Failed)
        128 - M/clmin (Failed)
        129 - M/clratio (Failed)
        130 - M/cut (Failed)
        131 - M/dim (Failed)
        132 - M/dim2 (Failed)
        133 - M/dim_expr (Failed)
        134 - M/faset (Failed)
        137 - M/interface (Failed)
        138 - M/interface2 (Failed)
        139 - M/interface3 (Failed)
        140 - M/mesh2dalgo_dela (Failed)
        141 - M/mesh2dalgo_fron (Failed)
        142 - M/mesh2dalgo_mead (Failed)
        143 - M/mesh2dalgo_netg (Failed)
        144 - M/mesh3dalgo_netggmne (Failed)
        145 - M/mesh3dalgo_netggmsh (Failed)
        146 - M/mesh3dalgo_netgnetg (Failed)
        147 - M/meshing (Failed)
        149 - M/meshqualdisexpr (Failed)
        150 - M/meshqualexpr (Failed)
        151 - M/meshqualmin (Failed)
        152 - M/nset1 (Failed)
        153 - M/nset2 (Failed)
        154 - M/nset3 (Failed)
        155 - M/order (Failed)
        156 - M/order_dim2 (Failed)
        161 - M/part_dim2 (Failed)
        162 - M/pl (Failed)
        163 - M/rcl (Failed)
        164 - M/rcl_expr (Failed)
        167 - M/remesh1 (Failed)
        168 - M/remesh2 (Failed)
        169 - M/remesh3 (Failed)
        181 - M/tesr_dim2 (Failed)
        184 - M/tess2ddim3 (Failed)
Errors while running CTest_

However when doing gmsh-4.9.5-source/build$ cmake .., the following description is :

_zirong@DESKTOP-D80F0JF:/mnt/c/Ubuntu_app_softwares/gmsh-4.9.5-source/build$ cmake ..
-- Found 64Bit
-- Found OpenMP
-- Found Eigen[contrib]
-- Found Mesh
-- Found Solver
-- Found Post
-- Found Plugins
-- Found Parser
-- Could NOT find OpenGL (missing: OPENGL_opengl_LIBRARY OPENGL_glx_LIBRARY OPENGL_INCLUDE_DIR)
-- Could NOT find X11 (missing: X11_X11_INCLUDE_PATH X11_X11_LIB)
-- Could NOT find FLTK (missing: FLTK_LIBRARIES FLTK_INCLUDE_DIR)
-- Found ONELAB
-- Found ONELABMetamodel
-- Found ANN[contrib]
-- Found ALGLIB[contrib]
-- Found DIntegration
-- Found OptHom
-- Found DomHex
-- Found QuadTri
-- Found Kbipack
-- Found GMP
-- Found MathEx[contrib]
-- Found Metis[contrib]
-- Found TetGen/BR
-- Found Voro++[contrib]
-- Found Blossom
-- Found Netgen
-- Found Bamg
-- HDF5: Using hdf5 compiler wrapper to determine C configuration
-- Found Gmm[contrib]
-- Found Hxt
-- Found QuadMeshingTools
-- Found WinslowUntangler
-- OCC CAF lib TKXDESTEP not Found
-- OCC CAF lib TKXDEIGES not Found
-- OCC CAF lib TKXCAF not Found
-- OCC CAF lib TKLCAF not Found
-- OCC CAF lib TKVCAF not Found
-- OCC CAF lib TKCAF not Found
-- OCC CAF lib TKV3d not Found
-- OCC CAF lib TKService not Found
-- OCC CAF lib TKCDF not Found
-- Found Dlopen
-- Found LinuxJoystick_

please help me how to deal with the ctest error for Neper.

Many thanks
Looking forward to hearing from you soon
Zirong

**Comment by rquey** [comment, marked as answer]
Did you actually compile Gmsh? Is it available at command line, as `gmsh`?

It is a good idea to install the FLTK dependency (then `cmake ..; make` again, in Gmsh).

**Comment by zirongzhai** [comment]
HI Romain:

 

I tried VM Ubuntu again and it works. Seems it is hard to implement Neper and FEPX in WSL.

 

Thanks for your advice and great work on that.

 

Kind regards

Zirong

 

发件人: Romain Quey <> 
发送时间: 2022年4月20日 22:40
收件人: rquey/neper ***@***.***>
抄送: Zirong Zhai ***@***.***>; Author ***@***.***>
主题: Re: [rquey/neper] Neper tests FAILED (Discussion #443)

 

Did you actually compile Gmsh? Is it available at command line, as gmsh?

It is a good idea to install the FLTK dependency (then cmake ..; make again, in Gmsh).
